front/src/actions/profile.js

```diff
@@ -175,7 +175,6 @@ function createActions(store) {
         });
         route('/dashboard/settings/user');
       } catch (e) {
-        console.log(e);
```

Comment:
Don't we prefer to log this has error (or warn)?

Reply:
I think we could just replace this with a console.error() yes 

Because as Gladys is an open-source product without tracking, we sometimes ask users to show the browser logs to debug (no sentry, no analytics tool here), and console errors are useful

Reply:
We try to avoid asking users for logs at all cost (not very user-friendly), but sometimes it's easier to find what's wrong ^^

Reply:
should this be `logger.error`?

Reply:
No, in front side, we don't have access to server logger, we use `console.error`

Reply:
Ah yes sorry - still new to the codebase. I'll change to console.error and add an eslint ignore and comment

Reply:
No need for an eslint ignore, console.error is allowed :)

Reply:
Ah yes sorry, didn't see your change. Looks good to me